canvas_modules/common-canvas/src/common-properties/panels/action-panel/action-panel.jsx

```diff
@@ -20,12 +20,19 @@ import { connect } from "react-redux";
 import classNames from "classnames";
 import * as ControlUtils from "./../../util/control-utils";
 import { STATES } from "./../../constants/constants";
+import { get } from "lodash";
 
 class ActionPanel extends Component {
 	render() {
 		const className = this.props.panel.className ? this.props.panel.className : "";
 		return (
-			<div className={classNames("properties-action-panel", { "hide": this.props.panelState === STATES.HIDDEN }, className)}
+			<div
+				className={classNames(
+					"properties-action-panel",
+					{ "hide": this.props.panelState === STATES.HIDDEN },
+					{ "properties-control-nested-panel": get(this.props.panel, "nestedPanel", false) },
```

Comment:
I think you can just use `this.props.panel.nestedPanel` since you already default this in the form generation.